indico/modules/events/management/controllers.py

```diff
@@ -178,3 +178,13 @@ def _update_session_coordinator_privs(self, form):
             log_msg = 'Session coordinator privilege changed to {}: {}'.format(form.data[priv_field],
                                                                                COORDINATOR_PRIV_TITLES[priv_field])
             self.event_new.log(EventLogRealm.management, EventLogKind.positive, 'Protection', log_msg, session.user)
+
+
+class RHMoveEvent(RHConferenceModifBase):
```

Comment:
I'd say bulk move should go into categories/controllers/management and single-event move into the events/management/controllers. The protection checks are different (moving events in a category requires category management rights, moving a specific event requires event management rights, and both of them require event creation rights in the target category)